### DISCUSSION TITLE
Explain the Parameters of AutoQueryEngine

### DISCUSSION THREAD

**Discussion by abrahimzaman360** [safevideo/autollm#219] [Q&A]

**Comment by abrahimzaman360** [comment]
You have added some new parameters, can you explain or define them in short what they do! If possible (:

**Comment by talhasarit** [comment]
which of them you want to know most @AbrahimZaman360 ?

**Reply by abrahimzaman360** [reply]
enable_qa_extract, refine prompt etc
